src/System.Text.Json/src/System/Text/Json/Serialization/Converters/JsonValueConverterGuid.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Text.Json.Serialization.Policies;
+
+namespace System.Text.Json.Serialization.Converters
+{
+    internal sealed class JsonValueConverterGuid : JsonValueConverter<Guid>
+    {
+        public override bool TryRead(Type valueType, ref Utf8JsonReader reader, out Guid value)
+        {
+            return reader.TryGetGuid(out value);
```

Comment:
Debug.Assert(valueType == typeof(Guid))?  (If that holds?)

Reply:
Yeah, this isn't actually used. I'll look at removing this parameter.